papermill/tests/test_engines.py

```diff
@@ -434,39 +434,50 @@ def test_nb_convert_engine_execute(self):
     def test_nb_convert_log_outputs(self):
         with patch.object(logger, 'info') as info_mock:
             with patch.object(logger, 'warning') as warning_mock:
-                with patch.object(NotebookExecutionManager, 'save'):
-                    NBClientEngine.execute_notebook(
-                        self.nb,
-                        'python',
-                        output_path='foo.ipynb',
-                        progress_bar=False,
-                        log_output=True,
-                    )
-                    info_mock.assert_has_calls(
-                        [
-                            call('Executing notebook with kernel: python'),
-                            call('Executing Cell 1---------------------------------------'),
-                            call('Ending Cell 1------------------------------------------'),
-                            call('Executing Cell 2---------------------------------------'),
-                            call('None\n'),
-                            call('Ending Cell 2------------------------------------------'),
-                        ]
-                    )
-                    warning_mock.is_not_called()
+                with patch.object(notebook_logger, 'info') as notebook_info_mock:
+                    with patch.object(notebook_logger, 'warning') as notebook_warning_mock:
+                        with patch.object(NotebookExecutionManager, 'save'):
```

Comment:
```suggestion
with patch.object(logger, 'info') as info_mock, patch.object(logger, 'warning') as warning_mock, patch.object(notebook_logger, 'info') as notebook_info_mock, patch.object(notebook_logger, 'warning') as notebook_warning_mock, patch.object(NotebookExecutionManager, 'save'):
```
consider reducing the depth